layouts/partials/content/meetup.html

```diff
@@ -0,0 +1,36 @@
+<div class="column is-one-third-desktop is-half-tablet">
+  <a {{ if .event_url }}href="{{ .event_url }}" rel="noopener" target="_blank"{{ end}}>
+    <div class="card card-equal-height">
+      {{ if .photo_url }}
+        <!-- Image -->
+        <div class="card-image">
+          <figure class="image">
+            <img src="{{ .photo_url }}" data-src="{{ .photo_url }}" alt="{{ .name }}" class="lozad card-image--figure">
```

Comment:
We'd want to use the high res version of the photo since right now, it's quite blurry

https://secure.meetupstatic.com/photos/event/b/8/8/global_480122952.jpeg

https://secure.meetupstatic.com/photos/event/b/8/8/highres_480122952.jpeg

Reply:
We'd also want a default image to indicate it's from Meetup.com - see https://techqueria.org/communities/

Reply:
https://gohugo.io/functions/replace/ would help here